src/libraries/Common/tests/System/FunctionPointerTests.cs

```diff
@@ -299,6 +332,68 @@ private unsafe class FunctionPointerHolder
 
             public class MyClass { }
             public struct MyStruct { }
+            private class PrivateClass { }
+        }
+
+        /// <summary>
+        /// A mock function pointer type used to test the abstract ContainsGenericParameters implementation.
```

Comment:
Move this test to src\libraries\System.Runtime\tests\System.Runtime.Tests\System\Type\TypeTests.cs. It is going to fix the build break.